.gitignore

```diff
@@ -202,6 +202,7 @@ fonts/.uuid
 *~
 .deps
 .dirstamp
+*.cfg-bak
```

Comment:
Can we treat wmllint's behavior of creating backup files as a bug, that has no place in a world with version-control tools? (That would probably need the default change to just linting, without making unexpected changes to files).

Reply:
That would probably require changing the behavior of the `-r` flag to `wmllint` as well, since it relies on those files

Reply:
Or at least have a flag to disable the backups behaviour.

Reply:
There are probably still plenty of UMC that are developed without the assistance of version control. They probably rely on this feature for so far as they actually use the tool.

Reply:
There's an option to remove the backup files instead: -c / --clean